compiler/src/symboltable.rs

```diff
@@ -680,11 +682,11 @@ impl SymbolTableBuilder {
                 self.scan_expressions(vals, context)?;
             }
             Subscript { a, b } => {
-                self.scan_expression(a, context)?;
+                self.scan_expression(a, &ExpressionContext::Load)?;
```

Comment:
The result that symtable analyze assign ast may be error.
e.g: 
```python
def f():
    sys.modules["test"] = test
```
![image](https://user-images.githubusercontent.com/17235904/87827258-1c6ca100-c8ad-11ea-82b8-1a109e00849a.png)
Identity get a assign context, and symtable think the variable scope is local.